code/modules/integrated_electronics/core/device.dm

```diff
@@ -0,0 +1,84 @@
+/obj/item/device/assembly/electronic_assembly
+	name = "electronic device"
+	desc = "It's a case for building electronics with. It can be attached to other small devices."
+	icon_state = "setup_device"
+	var/opened = 0
+
+	var/obj/item/device/electronic_assembly/device/EA
+
+/obj/item/device/assembly/electronic_assembly/New()
+	EA = new(src)
+	EA.holder = src
+	..()
+
+/obj/item/device/assembly/electronic_assembly/attackby(obj/item/I as obj, mob/user as mob)
+	if (iscrowbar(I) )
```

Comment:
This is the only time `iscrowbar()` is used. Instead of adding the proc, just do the istype check here.